src/database.ts

```diff
@@ -3346,6 +3349,91 @@ class Database extends common.GrpcServiceObject {
     return proxyStream as NodeJS.ReadableStream;
   }
 
+  /**
+   * Apply Blind Write of the Mutations
+   *
+   * writeAtLeastOnce(Blind Write) requests are not replay protected, meaning that it may apply mutations more
```

Comment:
```suggestion
   * writeAtLeastOnce writes mutations to Spanner using a single Commit RPC.
   * These requests are not replay protected, meaning that it may apply mutations more
```

Reply:
Done